go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/ASE/deb-simple-1
```

Comment:
should this be `github.com/esell/deb-simple` ?

Reply:
It should